libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -216,88 +128,15 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
-    // @Description{Copter, Blimp, Rover, Sub}: This parameter and function is not used by this vehicle. Always set to 0.
-    // @User: Advanced
-    AP_GROUPINFO("2_AUTOCAL",  16, AP_Airspeed, param[1].autocal, 0),
-
-    // @Param: 2_TUBE_ORDR
-    // @DisplayName: Control pitot tube order of 2nd airspeed sensor
-    // @Description: This parameter allows you to control whether the order in which the tubes are attached to your pitot tube matters. If you set this to 0 then the first (often the top) connector on the sensor needs to be the stagnation pressure (the pressure at the tip of the pitot tube). If set to 1 then the second (often the bottom) connector needs to be the stagnation pressure. If set to 2 (the default) then the airspeed driver will accept either order. The reason you may wish to specify the order is it will allow your airspeed sensor to detect if the aircraft is receiving excessive pressure on the static port compared to the stagnation port such as during a stall, which would otherwise be seen as a positive airspeed.
-    // @User: Advanced
-    // @Values: 0:Normal,1:Swapped,2:Auto Detect
-    AP_GROUPINFO("2_TUBE_ORDR",  17, AP_Airspeed, param[1].tube_order, 2),
-
-    // @Param: 2_SKIP_CAL
-    // @DisplayName: Skip airspeed offset calibration on startup for 2nd sensor
-    // @Description: This parameter allows you to skip airspeed offset calibration on startup, instead using the offset from the last calibration. This may be desirable if the offset variance between flights for your sensor is low and you want to avoid having to cover the pitot tube on each boot.
-    // @Values: 0:Disable,1:Enable
-    // @User: Advanced
-    AP_GROUPINFO("2_SKIP_CAL",  18, AP_Airspeed, param[1].skip_cal, 0),
-
-    // @Param: 2_PSI_RANGE
-    // @DisplayName: The PSI range of the device for 2nd sensor
-    // @Description: This parameter allows you to to set the PSI (pounds per square inch) range for your sensor. You should not change this unless you examine the datasheet for your device
-    // @User: Advanced
-    AP_GROUPINFO("2_PSI_RANGE",  19, AP_Airspeed, param[1].psi_range, PSI_RANGE_DEFAULT),
-
-    // @Param: 2_BUS
-    // @DisplayName: Airspeed I2C bus for 2nd sensor
-    // @Description: The bus number of the I2C bus to look for the sensor on
-    // @Values: 0:Bus0(internal),1:Bus1(external),2:Bus2(auxillary)
-    // @User: Advanced
-    AP_GROUPINFO("2_BUS",  20, AP_Airspeed, param[1].bus, 1),
+    // @Group: 1_
+    // @Path: AP_Airspeed_Params.cpp
+    AP_SUBGROUPINFO(param[0], "1_", 25, AP_Airspeed, AP_Airspeed_Params),
```

Comment:
```suggestion
    AP_SUBGROUPINFO(param[0], "_", 25, AP_Airspeed, AP_Airspeed_Params),
```

This would save us from invalidating a bunch of parameter files (and possibly breaking people's scripts), and keeps us consistent with the GPS, battery, and compass libraries. (Although I do agree if doing it from scratch again I would definitely prefer to see the various sensors all counting the same way).

Reply:
yeah, makes sense to remove the 1.  Hadn't thought of that 😅

Reply:
Done

Reply:
The diff's way smaller now as well, thanks!